### ISSUE TITLE
Add basic `Slack` integration.

### ISSUE THREAD

**Issue by ta2edchimp** [ta2edchimp/BattleReporter#60]
Push newly created BattleReports to a selected Slack channel using [`Incoming WebHooks`](https://slack.com/services/new/incoming-webhook).

**Comment by ta2edchimp** [comment]
Sample content for the payload to send to `Slack`:

```
{
    "text": "<http://battlereporter.dev/show/123|Battle: Us vs. Them> in Tama on 2015-02-26 13:37",
    "fields": [
        {
            "title": "posted by",
            "value": "User Name",
            "short": true
        }
    ],
    "color": "#F35A00"
}
```

should look like:
![https://api.slack.com/img/api/attachment_example_honeybadger.png]

**Comment by ta2edchimp** [comment]
Closed by v0.3.6
